CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ CHANGELOG
 
 **CHANGES**
 - Upgrade Slurm to version 24.05.8.
+- Support DCV for US Isolated Regions.
+- Support FSX for Lustre and Ontap in US Isolated region.
```

Comment:
nit: We should mention the regions where it is supported.
Will specify when iterating on the changelog as part of the release, no worries.